### PR TITLE
Darken social buttons for accessibility

### PR DESCRIPTION
**Pull request by loic-sharma**
For WCAG 2.1, web components must have a 3:1 contrast ratio. This darkens the social media icons on the display package page.

Before:

image

After:

image

Addresses https://github.com/NuGet/NuGetGallery/issues/8185

### CODE REVIEW

File: src/NuGetGallery/Content/gallery/img/twitter.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 400"><defs><style>.cls-1{fill:#109bf1;}.cls-2{fill:#fff;}.cls-3{fill:none;}</style></defs><title>Twitter_Logo_White-on-Blue</title><g id="Dark_Blue" data-name="Dark Blue"><rect class="cls-1" width="400" height="400"/></g><g id="Logo_FIXED" data-name="Logo — FIXED"><path class="cls-2" d="M153.62,301.59c94.34,0,145.94-78.16,145.94-145.94,0-2.22,0-4.43-.15-6.63A104.36,104.36,0,0,0,325,122.47a102.38,102.38,0,0,1-29.46,8.07,51.47,51.47,0,0,0,22.55-28.37,102.79,102.79,0,0,1-32.57,12.45,51.34,51.34,0,0,0-87.41,46.78A145.62,145.62,0,0,1,92.4,107.81a51.33,51.33,0,0,0,15.88,68.47A50.91,50.91,0,0,1,85,169.86c0,.21,0,.43,0,.65a51.31,51.31,0,0,0,41.15,50.28,51.21,51.21,0,0,1-23.16.88,51.35,51.35,0,0,0,47.92,35.62,102.92,102.92,0,0,1-63.7,22A104.41,104.41,0,0,1,75,278.55a145.21,145.21,0,0,0,78.62,23"/><rect class="cls-3" width="400" height="400"/></g></svg>
```
Comment (loic-sharma) [inline src/NuGetGallery/Content/gallery/img/twitter.svg:1]: The only thing that changed here is this part: `.cls-1{fill:#109bf1;}`

**Comment by joelverhagen** [comment]
We have fallback PNGs in most cases for SVG. Could you update those and ensure to minify the PNGs?

**Comment by loic-sharma** [comment]
> We have fallback PNGs in most cases for SVG. Could you update those and ensure to minify the PNGs?

Good catch! I updated the .pngs and tested the fallbacks when the svg links are broken.

**Comment by joelverhagen** [comment]
Please use lossless compression, such as with PNGGuantlet or a similar tool.
image

**Comment by loic-sharma** [comment]
I've remade the PNGs and added these [OneNote instructions](https://microsoft.sharepoint.com/teams/NuGet/_layouts/OneNote.aspx?id=%2Fteams%2FNuGet%2FTeam%2FNugetServer%2FNugetServerTeamNote&wd=target%28Operations.one%7CC674B49C-4CB7-4793-8890-A1E46E33FEB9%2FConvert%20SVG%20to%20PNG%20%28aka%20%22rasterizing%22%5C%29%7C9EADEFE9-840C-4A7E-BC03-DCB71335729F%2F%29onenote:https://microsoft.sharepoint.com/teams/NuGet/Team/NugetServer/NugetServerTeamNote/Operations.one#Convert%20SVG%20to%20PNG%20(aka%20%22rasterizing%22)&section-id={C674B49C-4CB7-4793-8890-A1E46E33FEB9}&page-id={9EADEFE9-840C-4A7E-BC03-DCB71335729F}&end) for my process.

**Review by joelverhagen — APPROVED** [review]

**Pull request merged**